{{cookiecutter.repo_name}}/src/components/PasswordField.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import { IconButton, InputAdornment } from '@material-ui/core';
+import Visibility from '@material-ui/icons/Visibility';
+import VisibilityOff from '@material-ui/icons/VisibilityOff';
+
+import { Field } from 'formik';
+import { TextField } from 'formik-material-ui';
+
+const PasswordField = ({ dataCy, ...rest }) => {
+    const [showPassword, setShowPassword] = useState(false);
+
+    const handleClickShowPassword = () => {
+        setShowPassword(prevShowPassword => !prevShowPassword);
+    };
+
+    const handleMouseDownPassword = event => {
+        event.preventDefault();
+    };
+    return (
+        <Field
+            component={TextField}
+            type={showPassword ? 'text' : 'password'}
+            data-cy={dataCy}
+            {...rest}
+            {% raw -%}InputProps={{
```

Comment:
Yeah i saw your note about the changing jinja to not look at {{}}. That would be a great improvement because this is the kind of thing that's difficult to format correctly AND get right for cookiecutter